src/input.rs

```diff
@@ -0,0 +1,9 @@
+//! Input processing functionality for Sprocket.
+
+pub mod yaml;
+
+// Re-export the main functionality from the yaml module
+pub use yaml::InputFormat;
+pub use yaml::get_json_file_path;
+// Removed unused export: pub use yaml::get_json_string;
+pub use yaml::parse_input_file;
```

Comment:
```suggestion
pub use yaml::get_json_file_path;
```
please commit this exactly

Reply:
`pub use yaml::get_json_file_path` is already exported @a-frantz .
I'm sorry to bother you but can you clarify it